### DISCUSSION TITLE
使用puppeteer打包报错

### DISCUSSION THREAD

**Discussion by wumingluren** [alex8088/electron-vite#536] [Q&A]
运行正常打包后运行报错。
使用puppeteer-core
先报错没有路径
const electronExePath = app.getPath('exe')
const browser = await puppeteer.launch({
executablePath: electronExePath
})
又报错无法启动

**Comment by alex8088** [comment]
路径问题，puppeteer-core 推荐打包到asar，相关可以参考 
- https://cn.electron-vite.org/guide/distribution#asar-存档局限性
- https://cn.electron-vite.org/guide/assets#app-asar-unpacked-文件路径问题

**Reply by wumingluren** [reply]
import puppeteer from 'puppeteer?asset&asarUnpack'
这样写也没有真的导入，但是至少打包后软件可以运行起来，现在打包出来的路径在resources/app.asar.unpacked/node_modules 里面。

puppeteer-core 推荐打包到asar 也没看懂要怎么弄。
大佬能给个例子吗？
